apps/public/content/docs/api/track.mdx

```diff
@@ -47,6 +47,10 @@ curl -X POST https://api.openpanel.dev/track \
 }'
 ```
 
+To track events for an identified user, add `profileId` to the payload.
+
+To specify a date/time for the tracked event (defaults to now), add `__timestamp` (formatted `YYYY-MM-DD HH:mm:ss` without tz) to properties.
```

Comment:
This is not exactly true. Should be `ISO 8601` format (`2025-06-04T08:08:30.202Z`). 

Or mention that the datetime must be UTC